src/app/models/indicator-query-opts.model.ts

```diff
@@ -7,8 +7,10 @@ export interface IndicatorQueryOpts {
     indicator: Indicator;
     city: City;
     scenario: Scenario;
-    climateModels?: ClimateModel[];
-    years?: string[];
-    time_aggregation?: string;
-    unit?: string;
+    params: {
```

Comment:
Hm these nested in params diverges from the API response: 

<img width="297" alt="screen shot 2017-08-09 at 5 10 29 pm" src="https://user-images.githubusercontent.com/10568752/29144106-b3913978-7d25-11e7-9731-510fc0c8013f.png">

Maybe you were looking at indicator.parameters ?